### PR TITLE
Fix test error of compressible agent

### PR DESCRIPTION
**Pull request by kevin666aa**
## Why are these changes needed?

Fix test error of compressible agent: "request_timeout" -> "timeout"

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by sonichi** [comment]
why are these tests run in the main openai test?

**Comment by kevin666aa** [comment]
Do we need to test all test files in `openai.yml` and ? 
```
          coverage run -a -m pytest test
```
Then it will be test again in contrib workflows.

**Comment by kevin666aa** [comment]
> why are these tests run in the main openai test?

Yes, that's the question, there is a `coverage run -a -m pytest test`

**Comment by sonichi** [comment]
we need to skip the contrib folder

**Comment by kevin666aa** [comment]
> we need to skip the contrib folder

Sure, that's what I thought, I will add it here

**Comment by sonichi** [comment]
The test still fails https://github.com/microsoft/autogen/actions/runs/6835278758/job/18588897636?pr=631

**Comment by kevin666aa** [comment]
> The test still fails https://github.com/microsoft/autogen/actions/runs/6835278758/job/18588897636?pr=631

Sorry, it is good now.

**Review by qingyun-wu — COMMENTED** [review]
LGTM. Thanks Yiran!

File: autogen/agentchat/contrib/compressible_agent.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def _set_compress_config(self, compress_config: Optional[Dict] = False):
             if compress_config.get("mode", "TERMINATE") not in allowed_modes:
                 raise ValueError(f"Invalid compression mode. Allowed values are: {', '.join(allowed_modes)}")
 
-            self.compress_config = self.DEFAULT_COMPRESS_CONFIG
+            self.compress_config = copy.deepcopy(self.DEFAULT_COMPRESS_CONFIG)
```
Comment (sonichi) [inline autogen/agentchat/contrib/compressible_agent.py:142]: do we need a deepcopy here?

Comment (kevin666aa) [reply autogen/agentchat/contrib/compressible_agent.py]: `self.compress_config = self.DEFAULT_COMPRESS_CONFIG.copy()` is good, changed it.

**Review by sonichi — APPROVED** [review]

**Pull request merged**